icubam/predicu/tests/test_export_data.py

```diff
@@ -18,17 +16,11 @@ def test_export_data(tmpdir, monkeypatch):
     make_monkeypatch_load_bedcounts(cached_data['bedcounts'])
   )
   import icubam.predicu.__main__
-  TestArgs = collections.namedtuple(
-    'TestArgs', [
-      'output_dir', 'api_key', 'max_date', 'icubam_host',
-      'spread_cum_jump_correction'
-    ]
-  )
-  test_args = TestArgs(
+  test_args = dict(
     output_dir=str(tmpdir),
     api_key='test_api_key',
```

Comment:
Is this thing hitting the /db/* API with a key to be able to get at the data?  How does it know what key?

Reply:
No I'm not using this CLI script. I only migrated it from argparse to click so it can be used as part of the same CLI group.

The added plotting scipt directly queries the DB in https://github.com/icubam/icubam/blob/9cf8f0d02755a231769abba9f6907a4781306f8b/icubam/predicu/analytics_server.py#L18